native/cocos/base/Utils.h

```diff
@@ -113,6 +113,12 @@ inline T alignTo(T size, T alignment) {
     return ((size - 1) / alignment + 1) * alignment;
 }
 
+// align to power of 2
+template <typename T, typename = typename std::enable_if<std::is_integral<T>::value>>
```

Comment:
What's the difference between align and alignTo?

Reply:
`align` usees bitwise operators will be more efficient. However `alignment` here must be power of 2.

Reply:
It is better:
- have a more meaningful name that can know the limitation of alignment
- please add static assert for alignment, make sure it is power of 2